extensions/cornerstone/src/init.js

```diff
@@ -284,10 +289,11 @@ export default async function init({
       contextMenuHandleClick
     );
 
-    eventTarget.removeEventListener(EVENTS.STACK_VIEWPORT_NEW_STACK, evt => {
-      const { element } = evt.detail;
-      utilities.stackPrefetch.disable(element);
-    });
+    // TODO - consider removing the callback when all elements are gone
+    // eventTarget.removeEventListener(
+    //   EVENTS.STACK_VIEWPORT_NEW_STACK,
+    //   newStackCallback
+    // );
```

Comment:
I believe we need this back.

Reply:
It was a no-op previously because the removeEventListener got called with a new object, so it was never in the list, and never would remove anything.  So, it isn't any worse than the previous behaviour, although I agree that eventually figuring out if no viewports are displayed to remove the listener is a good idea.  Or maybe not - just leaving it indefinitely and ensuring that it doesn't do anything when the viewport is another type also works.